src/java-api-bindings/scripts/install_dependencies_and_build.sh

```diff
@@ -91,35 +95,78 @@ for OPTS; do
         echo "Javacpp-presets branch tag set to: ${JAVACPP_BRANCH_TAG}"
         shift 2
         ;;
+        --include-developer-tools-server) 
+        INCLUDE_DEVELOPER_TOOLS_SERVER=1
+        echo "Including developer tools server C++ bindings"
+        ;;
     esac
 done
 set -x
 
 # Install jdk and maven
+rm -r ${BUILD_HOME}
 mkdir -p ${BUILD_HOME}
 cd ${BUILD_HOME}
 apt update && apt install -y openjdk-11-jdk
 wget https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz
 tar zxvf apache-maven-${MAVEN_VERSION}-bin.tar.gz
-MAVEN_PATH=${BUILD_HOME}/apache-maven-${MAVEN_VERSION}/bin/mvn
+export PATH=$PATH:$PWD/apache-maven-${MAVEN_VERSION}/bin/
 
+# Build static libraries and copy wrapper includes to triton home
 # Copy necessary tritonserver .h files so the bindings can be generated
 mkdir -p ${TRITON_HOME}/lib/
 cd ${BUILD_HOME}
 CORE_BRANCH=${CORE_BRANCH:="https://github.com/triton-inference-server/core.git"}
 git clone --single-branch --depth=1 -b ${CORE_BRANCH_TAG} ${CORE_BRANCH}
 cp -r core/include ${TRITON_HOME}/include
 
+if [ ${INCLUDE_DEVELOPER_TOOLS_SERVER} -eq 1 ]; then
+    # install cmake and rapidjson
+    wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | \
+        gpg --dearmor - |  \
+        tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null && \
+        apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' && \
+        apt-get update && \
+        apt-get install -y --no-install-recommends \
+            cmake-data=${CMAKE_VERSION}-0kitware1ubuntu20.04.1 \
+            cmake=${CMAKE_VERSION}-0kitware1ubuntu20.04.1 \
+            rapidjson-dev
+
+    git clone --single-branch --depth=1 -b ${TOOLS_BRANCH_TAG} ${TOOLS_BRANCH} 
+    cd developer_tools/server
+    rm -r build 
+    mkdir build && cd build
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_TEST=ON -DTRITON_ENABLE_EXAMPLES=ON -DTRITON_BUILD_STATIC_LIBRARY=OFF .. 
+    make -j"$(grep -c ^processor /proc/cpuinfo)" install
+    # Copy dynamic library to triton home
+    cp ${BUILD_HOME}/developer_tools/server/build/install/lib/libtritondevelopertoolsserver.so ${TRITON_HOME}/lib/.
+
+    rm -r ${TRITON_HOME}/include/triton/developer_tools
```

Comment:
Would you be able to add variables here for the repeated paths? Could be helpful for readability.
e.g. for {BUILD_HOME}/developer_tools/server and {TRITON_HOME}/include/triton/developer_tools.